lib/wallet/bench/db-bench.hs

```diff
@@ -774,6 +777,22 @@ withCleanDB tr f g = perRunEnvWithCleanup setup (dbDown . fst) $
         x <- f dbFresh
         pure (be , x)
 
+-- | Put a sequence of checkpoints into the database.
```

Comment:
I propose to move this into the library and leave the db-bench as dumb as possible. (Importing the dbvar library is also failing the compilation btw)